app/services/hooks/useMyShifts.ts

```diff
@@ -2,18 +2,18 @@ import { useQuery } from "@tanstack/react-query"
 
 import { eventApi } from "../api/eventApi"
 
-export const useEvents = () => {
+export const useMyShifts = () => {
   const {
-    data: events,
+    data: myShifts,
     error,
     isPending,
     isFetching,
   } = useQuery({
-    queryKey: ["events"],
+    queryKey: ["myShifts"],
```

Comment:
All our query keys follow kebab case
```suggestion
    queryKey: ["my-shifts"],
```